From fe9333ce1d69e292e14144a61ff88a23ffed0053 Mon Sep 17 00:00:00 2001
From: Chris Renshaw <osm0sis@outlook.com>
Date: Thu, 22 Sep 2016 16:17:05 -0300
Subject: [PATCH] strings: Correct output to be decimal per the help, instead of octal
 Most smaller/incomplete implementations of strings output decimal by default,
 so best to correct that behavior rather than just change the help to say octal
 (it would be great to get full POSIX --radix at some point, to give the
 output in any desired format, but this will do for now)

---
 miscutils/strings.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/miscutils/strings.c b/miscutils/strings.c
index 9f50182..cc4ad9a 100644
--- a/miscutils/strings.c
+++ b/miscutils/strings.c
@@ -68,7 +68,7 @@ int strings_main(int argc UNUSED_PARAM, char **argv)
 							printf(fmt, *argv);
 						}
 						if (option_mask32 & PRINT_OFFSET) {
-							printf("%7"OFF_FMT"o ", offset - n);
+							printf("%7"OFF_FMT"d ", offset - n);
 						}
 						fputs(string, stdout);
 					}
-- 
2.8.3

